packages/dev-utils/src/matchers.ts

```diff
@@ -9,7 +9,8 @@ declare global {
     }
   }
 }
-jest.setTimeout(10000)
+
+jest.setTimeout(10_000)
```

Comment:
turns out this is actually useful, a lot of our tests timeout after 5 seconds (default of jest)